boards/riscv32/rv32m1_vega/doc/index.rst

```diff
@@ -170,8 +170,8 @@ Pin   Port/bit   Function
 12    B/4        SPI0_SCK
 14    -          GND
 16    -          AREF
-18    C/9        I2C2_SDA
-20    C/10       I2C2_SCL
+18    C/9        I2C0_SDA
```

Comment:
That's not what the schematic says. Are you sure?

Reply:
100% sure. In my schematic it also labelled as such (and I tested it with my soon to be submitted RV32M1 LPI2C driver).

Reply:
Hrm. @MaureenHelm, looks like there is a schematic bug in the docs package from the only release so far: here https://github.com/open-isa-org/open-isa.org/releases

@henrikbrixandersen where did you get your schematic?

Reply:
Here's where I got this information from in my schematic, on sheet 6.

![i2c](https://user-images.githubusercontent.com/35088/55754901-f0584400-5a0a-11e9-91ff-a9822940f9ae.png)

Reply:
![image](https://user-images.githubusercontent.com/1076226/55754982-2965d800-5a4e-11e9-80d1-d175163630f5.png)

I checked the other part of the schematic ;-)

Reply:
Yup, looks like a bug in the schematic. The SoC hardware reference manual (RV32M1RM_Rev1.1.pdf) shows PTC9/10 to have LPI2C0 signals routed into the pinmux.